chart/openfaas/templates/gateway.yaml

```diff
@@ -67,6 +67,10 @@ spec:
           value: "nats.{{ .Release.Namespace }}.svc.cluster.local."
         - name: faas_nats_port
           value: "4222"
+        - name: direct_functions
+          value: "true"
+        - name: direct_functions_suffix
+          value: "openfaas-fn.svc.cluster.local."
```

Comment:
The namespace for functions is "templated" / injected by helm so you can't use this value here since it's hard-coded. Try to see if you can find where we use the function namespace in faas-netesd and copy the approach.